Libraries/Microsoft.Teams.Extensions/Microsoft.Teams.Extensions.Configuration/Microsoft.Teams.Apps.Extensions/TeamsSettings.cs

```diff
@@ -24,7 +25,8 @@ public AppOptions Apply(AppOptions? options = null)
         return options;
     }
 
-    public bool IsNotEmpty() {
+    public bool IsNotEmpty()
```

Comment:
I probably forgot to remove this from my PR - it's been replaced with `Empty`. Do you mind removing it?